client/src/components/WrappedTicket/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link, Route } from 'react-router-dom'
+import './style.css'
+const WrappedTicket = (props) => {
+    console.log(props)
+    return (
+        <div className="wrapped-ticket-container">
+            <div> {props.wrappedTickets ? props.wrappedTickets.map(wrappedticket =>
+                <Route>
+                    <Link to={`/ticket/${wrappedticket.id}`} className="wrapped-ticket-container__link">
+                        <div className="wrapped-ticket-container__wrappedticket" >
+                            <div className="wrapped-ticket-container__check-email-container"><input type="checkbox" className="wrapped-ticket-container__checkbox" />
+                                {wrappedticket.email}
+                            </div>
+                            <p>{wrappedticket.subject}</p>
+                            <p className="wrapped-ticket-container__wrappedticket__ticket-descreption">{wrappedticket.description.substring(0, 49)}...</p>
```

Comment:
Why are we using a substring for the description? 
Some descriptions will be more than 49 characters, others will be less.

Reply:
to make all description view as the same at all time.